src/components/NavBar/NavBar.tsx

```diff
@@ -13,14 +13,13 @@ interface Props {
   /**
    * Set the backgound color of the `NavBar`
    */
-  backgroundColor: string;
+  backgroundColor?: string;
   /** Height Override */
   height?: string;
-  /** Display Gymnasium Logo
-   * To display, use `left` or `right`
-   * To hide, don't specify this prop or use `no`
-   */
-  displayLogo?: 'left' | 'right' | 'no';
+  /** Show Gymnasium Logo */
+  showLogo?: boolean;
+  /** The position of the logo (if shown) */
+  logoPosition?: 'left' | 'right';
```

Comment:
@alihamoody the logo will never be placed to the right :) Just left or nothing.  The more pertinent question here is how the navbar itself will be aligned otherwise, I think.

Take a look at the 4 bars here:

![image](https://user-images.githubusercontent.com/1844496/65458615-f8226e00-de1b-11e9-9644-b75b36ce7875.png)

1. has no logo, but has right justified content
1. has a logo and right justified content
1. again no logo, left justified content
1. no logo, left justified content

Reply:
Is there somewhere where you're seeing a logo on the right hand side?

Reply:
@mbifulco 
I'm aware the logo doesn't show on the right in the gymnasium.com. 
I included the **capability** to make it appear on the right, as UI library that could be used for other purposes. 
However, now I understand this is not desirable and I will remove it.
No worries :) 

This means the NavBar will be the same as the `SecondaryNavigation` that is required in another ticket/issue #2.